src/vis.ts

```diff
@@ -1,38 +1,63 @@
 import * as THREE from 'three';
-import * as plane from './plane';
+import {Plane} from './plane';
 import {OrbitControls} from 'three/examples/jsm/controls/OrbitControls';
 
 export class Vis {
   renderer: THREE.WebGLRenderer;
```

Comment:
Any properties that aren't part of the API of the class should be marked `private`. I think that's all of the properties of this class because you provide methods like `addAircraftModels` to access them (and it probably applies to some properties in other classes).

Making properties `private` means developers don't need to worry about weird things happening and messing up their assumptions -- i.e. "what if `scene` is set to `null` between the constructor and `drawFrame`" or "what if someone on another thread changes the valuable of a property when I'm halfway through a calculation using that property".

What I typically see is that either a class will have all its properties be public and it won't do any work (i.e. it only exists to bundle together variables) OR a class will have its properties all be private in which case it will primarily be concerned with implementing logic.